crates/context/config/src/client/env/config/query/members_revision.rs

```diff
@@ -63,3 +66,19 @@ impl Method<Starknet> for MembersRevisionRequest {
         Ok(revision)
     }
 }
+
+impl Method<Icp> for MembersRevisionRequest {
+    type Returns = Revision;
+
+    const METHOD: &'static str = "members_revision";
+
+    fn encode(self) -> eyre::Result<Vec<u8>> {
+        let context_id: ICContextId = (*self.context_id).rt()?;
```

Comment:
```suggestion
        let context_id: ICContextId = self.context_id.rt()?;
```

are these initial derefs necessary?

Reply:
Works, updated in all cases